### ISSUE TITLE
Clarity of glossary vs in-line definition items

### ISSUE THREAD

**Issue by ehuelsmann** [ehuelsmann/ledgersmb-book#34]
In chapter 3, there is:

https://github.com/ehuelsmann/ledgersmb-book/blob/ab7dcd464828558ba3895d2e62f84df0d97520a5/part-overview.tex#L359-L372

which seems exactly the definitions we'd want to add to the glossary. @neilt: The question is: when to put them like this (and do we then include them in the glossary still) and when to put them in the glossary without putting them in-line.

**Comment by neilt** [comment]
What I have wondered is why does the book not have an index?  I think a lot of items including these could just go in the index.

**Comment by ehuelsmann** [comment]
The book actually had an index, but it there was only 1 term, which didn't make it all too useful. With more terms, the usefulness of the index is likely to increase.

**Comment by neilt** [comment]
Do you want me to move these to the glossary?  That seems the right approach.

**Comment by neilt** [comment]
In the PDF environment, glossary entries are not visibly annotated as links.  For a printed book that makes sense.  But for on-line PDF I am not so sure.

Any thoughts?

**Comment by ehuelsmann** [comment]
Agreed: the glossary items should be recognizable in some shape or form.

**Comment by neilt** [comment]
> Agreed: the glossary items should be recognizable in some shape or form.

Ok, I'll look into it and make a proposal.

**Comment by ehuelsmann** [comment]
Instead of putting them in the glossary, we could put them in the index: that way the explanation remains part of the story line. I think it does make sense to do that, because that's what the chapter is about. However, by putting it in the index, people with less linear learning patterns at least can find the explanations...

**Comment by ehuelsmann** [comment]
By the way, the rendering of the definition list (on my browser) is pretty horrible:

![2023-04-26_21-33](https://user-images.githubusercontent.com/2326559/234683785-2311e0ff-3767-40bd-babd-b42a0757b4ff.png)

**Comment by neilt** [comment]
Something looks off.  Here is mine.
<img width="658" alt="Screenshot 2023-04-26 at 2 36 37 PM" src="https://user-images.githubusercontent.com/1809796/234684210-46811abd-bed1-4be8-b0bd-cc7abd3a57f2.png">

**Comment by neilt** [comment]
I concur, I'll look for a fix.

**Comment by neilt** [comment]
The whole point of using that list type was to start the text in the same spot, like it does in the PDF.  There are a number of issues with LatTeXML that I need to sort through.

**Comment by neilt** [comment]
> Instead of putting them in the glossary, we could put them in the index: that way the explanation remains part of the story line. I think it does make sense to do that, because that's what the chapter is about. However, by putting it in the index, people with less linear learning patterns at least can find the explanations...

OK, I'll add them to the index.

**Comment by ehuelsmann** [comment]
> Something looks off. Here is mine.
>
It looks like yours doesn't apply ltx-book.css. Mine does. (I'm looking at
it from the book website, btw)
